mobify/platform-scaffold
# PLATT-43: nearby widget to PDP

Add Nearby Widget to PDP

 **JIRA**: https://mobify.atlassian.net/browse/PLATT-43
 **Linked PRs**: https://github.com/mobify/platform-scaffold/pull/349

## Changes
- add Nearby Widget to PDP

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Go to one of PDP
- Make sure that nearby widget script only being added on PDP, it should not be added on any other pages than PDP.
